src/Graphics/OpenGLContext/mupen64plus/mupen64plus_DisplayWindow.cpp

```diff
@@ -173,7 +179,32 @@ bool DisplayWindowMupen64plus::_resizeWindow()
 
 void DisplayWindowMupen64plus::_changeWindow()
 {
+#ifdef GLIDENUI
```

Comment:
I think it would be nice to have a separate define for GLIDENUI_SUPPORTS_FULLSCREEN or something like that. Full screen resolution detection is pretty platform specific and might not work well on all frontends.

Reply:
I think it'd be saner if gliden64 would check if that function returns `M64ERR_UNSUPPORTED` and if it does, fall back to the original code

Reply:
Yes, in ```DisplayWindowMupen64plus::_start()``` you could try to execute ```FunctionWrapper::CoreVideo_SetVideoModeWithRate```, if that returns ```M64ERR_UNSUPPORTED```, that tells you that the frontend doesn't support full screen resolution setting, and the rest of the code should execute based on that knowledge

Reply:
Implemented.